bin/build_breakpad.sh

```diff
@@ -54,23 +57,30 @@ else
 fi
 
 cd src
-git checkout "$BREAKPAD_REV"
+git checkout --force "$BREAKPAD_REV"
 gclient sync
 
 echo ">>> using breakpad version: $(git rev-parse HEAD)"
 
+pwd
+# Apply local patches
+for p in ${BREAKPAD_PATCHES}/*.patch; do
+    echo "Applying $p"
+    if ! cat $p | patch -p1; then
+      echo "Failed to apply $p"
+      exit 1
+    fi
+done
+
 mkdir -p "${PREFIX}"
 rsync -a --exclude="*.git" ./src "${PREFIX}"/
 
-# NOTE(willkg): Swap these when you're building a pgo profile
-export PGOFLAGS="-fprofile-use=/app/pgo_profile/"
-# export PGOFLAGS="-fprofile-generate=/app/pgo_profile/"
-
 # Configure breakpad for building
-export CXXFLAGS="-g -flto -O3 ${PGOFLAGS}"
-./configure --prefix="${PREFIX}"
+OPTFLAGS="-O3 -flto ${PGOFLAGS} -Wno-coverage-mismatch -Wno-missing-profile"
```

Comment:
I noticed that `PGOFLAGS` is coming from the caller script which got me to wonder what was set in this script. I think this script should be doing `set -u` so unset variables error out. I can add that later or you could add it in this PR--either is fine.